### PR TITLE
PR #763 untangled:  Add consideration of browser permissions framework for extension processing

### PR DESCRIPTION
**Pull request by emlun**
This is what I believe to be the intended changes of #763, with untangled Git history. @gmandyam please verify.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/771.html" title="Last updated on Feb 5, 2018, 2:26 PM GMT (f0ad6c7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/771/9d5609d...f0ad6c7.html" title="Last updated on Feb 5, 2018, 2:26 PM GMT (f0ad6c7)">Diff</a>

### CODE REVIEW

**Comment by gmandyam** [comment]
@emlun 

Thanks for doing this.  It works for me.  I had this section under Sec Cons, but Privacy Cons seems to be more appropriate.

Approved.

**Comment by emlun** [comment]
Oops, that was an oversight on my part. I think its original placement under Sec Cons is more appropriate, do you object to moving it there?

**Comment by gmandyam** [comment]
@emlun 

No - please do so.

**Comment by equalsJeffH** [comment]
fixes #133

File: index.bs [hunk 1]
```diff
@@ -4467,6 +4467,14 @@ the wrong [=credential ID=], or if an attacker intercepts and manipulates the [=
 [=credential public key=] with which to verify the returned signed [=authenticator data=]
 (a.k.a., [=assertion=]), and thus the interaction would end in an error.
 
+
+## Browser Permissions Framework and Extensions ## {#browser-permissions-framework-extensions}
+
+The Web Authentication API should leverage the browser permissions framework as much as possible when obtaining user permissions for
```
Comment (equalsJeffH) [inline index.bs]: suggest:
s/The //
s/ API / API implementations /

File: index.bs [hunk 2]
```diff
@@ -4467,6 +4467,14 @@ the wrong [=credential ID=], or if an attacker intercepts and manipulates the [=
...
+certain extensions.  An example is the location extension (see [[#sctn-location-extension]]), which should make use of the existing browser permissions framework for the
```
Comment (equalsJeffH) [inline index.bs]: suggest:
s/ which / implementations of which /

**Comment by emlun** [comment]
@gmandyam Will you continue the work on this?

**Comment by gmandyam** [comment]
@emlun 

It would be easier (for me) if you incorporated the editorial changes requested by @equalsJeffH.  After that, I think we can merge this PR.  Afterwards, I can address additional comments that will come up during CR wide review.  Thanks.

**Comment by emlun** [comment]
Since @selfissued, @equalsJeffH and @gmandyam have all approved this PR, I'll merge it if I see no objections before 2018-02-06T10:00+01:00.

**Comment by nadalin** [comment]
@emlun Can you merge ?

**Pull request merged**